src/mappings.go

```diff
@@ -376,17 +376,21 @@ func (maps Mappings) UnlinkAll(repo abspath.AbsPath) error {
 }
 
 func (maps Mappings) ActualLinks(repo abspath.AbsPath) ([]PathLink, error) {
-	ret := make([]PathLink, 0, len(maps))
+	m := map[PathLink]bool{}
```

Comment:
```suggestion
	m := map[PathLink]struct{}{}
```

Would you use `struct{}` for empty value of map? It will not consume space.

Reply:
Definitely you are right. Thanks.